app/src/main/java/org/mozilla/fenix/settings/logins/fragment/LoginDetailFragment.kt

```diff
@@ -169,11 +169,7 @@ class LoginDetailFragment : Fragment(R.layout.fragment_login_detail) {
     }
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
-        if (FeatureFlags.loginsEdit) {
-            inflater.inflate(R.menu.login_options_menu, menu)
-        } else {
-            inflater.inflate(R.menu.login_delete, menu)
```

Comment:
Can you also delete this menu resource, as its no longer used?

Reply:
Done 👍🏻